### DISCUSSION TITLE
Is there a prawn equivalent of z-index?

### DISCUSSION THREAD

**Discussion by watsonchang15** [prawnpdf/prawn#1291] [General] [answered]
Hello!

Wondering, is there a prawn way for ordering elements to the front or back? I'm trying to draw a line but it gets rendered behind an existing element. I would like it to be in front by default. Oddly enough, the line shows up sometimes depending on the PDF application. It shows in adobe acrobat but not in preview (the OSX app).

Thanks all.

**Comment by pointlessone** [comment, marked as answer]
No, there's no analogue of CSS z-index or depth buffer. You have to pre-sort your drawing operations yourself. First draws are the "farthest". In order to get your line on top of everything else it has to be drawn last.

**Comment by watsonchang15** [comment]
Got it, thank you 🙏
